dependencies.gradle

```diff
@@ -1,18 +1,18 @@
 ext.versions = [
         minSdk               : 14,
-        targetSdk            : 22,
+        targetSdk            : 23,
         compileSdk           : 28,
         gradle               : '4.6',
 
-        androidTools         : "3.2.0",
+        androidTools         : "3.6.3",
         ktlint               : "0.28.0",
         ktlintGradle         : "5.1.0",
         detekt               : "1.0.0.RC6-4",
         jacoco               : "0.8.2",
 
         androidSupport       : "22.2.1",
```

Comment:
You can change this to:
```
androidSupport            : "1.0.0",
androidSupportDesign      : "1.0.0",
androidSupportAnnotations : "22.2.1",
```
And use this vars in `ext.androidSupport`

Reply:
Ah OK, I see now, thank you!
Moved all dependencies to dependencies.gradle, with separated versions.